tests/test_tekton.py

```diff
@@ -305,7 +305,7 @@ def test_start_pipeline(self, openshift, expected_request_body_pipeline_run,
         responses.add(
             responses.POST,
             f'https://openshift.testing/apis/tekton.dev/v1beta1/namespaces/{TEST_OCP_NAMESPACE}/pipelineruns', # noqa E501
-            match=[responses.json_params_matcher(expected_request_body)],
```

Comment:
Is this ok to not check request body? Content of request is dynamic.

I'm looking for solution to generate expected body